rsi/__main__.py

```diff
@@ -73,3 +88,27 @@ def new_rsi(loc: Path,
     rsi.write(loc, make_parents)
 
     return 0
+
+
+def web_rsi(url: str,
+            output: Path,
+            rsi_license: Optional[str] = "",
+            splitter: Optional[str] = "",
+            indents: Optional[int] = None) -> int:
+
+    if isinstance(indents, str):
```

Comment:
This violates the types. You should be doing an int conversion *before* calling `web_rsi`.